requirements.lock

```diff
@@ -1,4 +1,4 @@
-adal==1.2.1
+adal==1.2.2
```

Comment:
Since you haven't updated `requirements.txt`, I'd remove this file from the PR - we can update dependencies in a separate PR.

Reply:
Same with `package-lock.json`